site/viamguides/src/home-assistant/home-assistant.md

```diff
@@ -0,0 +1,190 @@
+
+![Home Assistant notification of a person detected](assets/ha-detection-notification.png)
+
+### What You’ll Build 
+- a machine running a local ML model
+- a smart home automation for people detection notifications
+
+### Prerequisites
+- A computer with MacOS, Windows, or Linux to flash your Raspberry Pi and configure the device's components using the Viam app
+- Hardware and supplies:
+  - 1 - [Raspberry Pi 5](https://www.amazon.com/Raspberry-Single-2-4GHz-Quad-core-Cortex-A76/dp/B0CLV7DFD2)
+    - Follow the [Raspberry Pi setup guide](https://docs.viam.com/installation/prepare/rpi-setup/) to make sure your Pi is flashed with a Viam-compatible operating system, and that you are able to SSH into it.
+  - 1 - microSD card to use with your Pi
+  - 1 - power supply for your Pi
+
+### What You’ll Need 
+- All the hardware components listed in prerequisites.
+- A Home Assistant instance with a [camera integration](https://www.home-assistant.io/integrations/#camera) and [HACS](https://hacs.xyz/docs/use/) installed 
+- Sign up for a free Viam account, and then [sign in](https://app.viam.com/fleet/dashboard) to the Viam app
+
+### What You’ll Learn 
+- How to use a vision service in Viam
+- How to install a community integration in Home Assistant
+- How to configure an automation in Home Assistant
+
+### Watch the Video
+
+See a demonstration and overview of the integration in this video.
+
+<!-- <video id=""></video> -->
+
+<!-- ------------------------ -->
+
+## Configure your machine
+Duration: 3
+
+### Create your machine
+
+1. In [the Viam app](https://app.viam.com/fleet/dashboard) under the **LOCATIONS** tab, create a machine by typing in a name and clicking **Add machine**.
+   ![add machine](assets/viam-add-machine.png)
+1. Click **View setup instructions**.
+   ![setup instructions](assets/viam-machine-setup.png)
+1. To install `viam-server` on the Raspberry Pi device that you want to use to communicate with and control your webcam, select the `Linux / Aarch64` platform for the Raspberry Pi, and leave your installation method as [`viam-agent`](https://docs.viam.com/how-tos/provision-setup/#install-viam-agent).
+   ![select platform](assets/viam-machine-install.png)
+1. Use the `viam-agent` to download and install `viam-server` on your Raspberry Pi. Follow the instructions to run the command provided in the setup instructions from the SSH prompt of your Raspberry Pi.
+   ![installation agent](assets/viam-agent.png)
+1. The setup page will indicate when the machine is successfully connected.
+   ![successful toast](assets/viam-machine-connected.png)
+
+### Add an ML model service
+
+1. In the Viam app, click the **+** icon in the left-hand menu and select **Service**, and then `ML model`.
+   ![select ML model](assets/viam-config-ml-model.png)
+1. Search for a module called `TFLite CPU`. Then click **Add module**, and **Create** a new vision service called `mlmodel-1`.
```

Comment:
TFLite (aka LiteRT) is popular. Would it be worth including a mention in the What You'll Learn/What You've Learned section, and linking to a page like this? https://ai.google.dev/edge/litert Even though we've abstracted the implementation details, it's good to reinforce what they're learning to use.